packages/web-react/src/types/shared/inputs.ts

```diff
@@ -1,6 +1,7 @@
 import type { ChangeEventHandler, KeyboardEventHandler } from 'react';
 
-export type ValidationState = 'success' | 'warning' | 'error';
+/* Deprecated (#DS-590) */
+export type ValidationState = 'success' | 'warning' | 'danger' | 'error';
```

Comment:
'success' | 'warning' | 'danger' should be in types/shared/dictionaries and here just extended with `error`. Please fix this.

Reply:
done